interfaces/automated-testing/scd/design/strategic-coordination-interoperability/nominal-test.plantuml

```diff
@@ -0,0 +1,28 @@
+@startuml
+center header Strategic Coordination Interoperability
+title Nominal test
+
+participant "Test Driver" as TD
+participant "USS1 under test" as USS1
+participant "USS2 under test" as USS2
+participant "DSS"
+
+== TD injects via USS1 non-conflicting Operational Intent 1 ==
+
+TD --> USS1: PUT /inject/operational_intent
+TD <-- USS1: 200 OK - Operational Intent 1 created
+
+TD --> DSS: POST /dss/v1/operational_intent_references/query\nwith area_of_interest encompassing Operational Intent 1
+TD <-- DSS: 200 OK - returns Operational Intent Reference 1 with entityid1 managed by USS1
+
+TD --> USS1: GET /uss/v1/operational_intents/{entityid1}
+TD <-- USS1: 200 OK - returns Operational Intent 1
+
+...
+
+== TD injects via USS2 a conflicting Operational Intent (time, altitude, priority and area) ==
+
+TD --> USS2: PUT /inject/operational_intent
+TD <-- USS2: 409 Conflict - Operational Intent 2 rejected
```

Comment:
I would suggest that the injection API return 200 here -- there was no error on the USS side; it appropriately decided that the requested flight could not be planned and gracefully indicated that fact to the "user" (qualifier).  Instead, we can have a response structure that contains information for a number of possible outcomes, including both "flight was successfully planned" and "flight could not be planned due to a conflict".